enterprise/frontend/src/metabase-enterprise/audit_app/containers/AuditTable.jsx

```diff
@@ -39,97 +26,60 @@ const mapDispatchToProps = {
 
 const DEFAULT_PAGE_SIZE = 100;
 
-@connect(
-  mapStateToProps,
-  mapDispatchToProps,
-)
-export default class AuditTable extends React.Component {
-  props: Props;
-  state: State = {
-    page: 0,
-    hasMorePages: false,
-  };
-
-  static defaultProps = {
-    pageSize: DEFAULT_PAGE_SIZE,
-  };
-
-  render() {
-    const { metadata, table, onChangeLocation, pageSize } = this.props;
-    const { page, hasMorePages } = this.state;
-
-    const card = chain(table.card)
-      .assoc("display", "audit-table")
-      .assocIn(["dataset_query", "limit"], pageSize)
-      .assocIn(["dataset_query", "offset"], pageSize * page)
-      .value();
+AuditTable.defaultProps = {
+  pageSize: DEFAULT_PAGE_SIZE,
+};
```

Comment:
nip: looks like `defaultProps` are going to be deprecated for functional components at some point (see [here](https://github.com/metabase/metabase/issues/16224#issuecomment-849019047)). Should we replace that with function default arguments right away?

Reply:
Fixed that, thanks for pointing it out